app/views/user_mailer/fake_submission_rejected.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= stylesheet_link_tag 'emails', media: 'all' %>
+
+<table class='padded-email'>
+  <tr>
+    <td>
+      <table class='email-content' align='left'>
+        <tr>
+          <td>
+            <%= render 'shared/email/email_header' %>
+          </td>
+        </tr>
+        <tr>
+          <td class='email-sub-title'>
+            <p>Dear <%= @user.name %>,</p>
+            <p>Thank you for submitting your work to Artsy.</p>
+            <p>After extensive research, we are unable to find a record of the artwork and attribute it to the style of painting. We are therefore unable to offer it for sale through Artsy at this time.</p>
```

Comment:
The use of painting seems weird here, since not all submissions are paintings. I've asked in [Slack](https://artsy.slack.com/archives/C01B2P6LJUU/p1630510596087800) about it.

Reply:
Let's change that text to 

> After extensive research, we are unable to find a record of the artwork and attribute it to the artist. We are therefore unable to offer it for sale through Artsy at this time.

based on the Slack conversation.

Reply:
Changed